src/components/MatchItem/Avatars.tsx

```diff
@@ -0,0 +1,73 @@
+import { AvatarGroup, Chip, Avatar, Badge } from '@mui/material'
+import { Participant } from '../../types/Match'
+
+export const Avatars = ({ participants }: { participants: Participant[] }) => {
+  const participantsByScore = participants.sort((a, b) => b.score - a.score)
+
+  const participant = participantsByScore.map((participant) => {
+    if (participant.isWinner) {
+      return (
+        <>
+          <Chip
+            sx={{ marginTop: '2px', marginRight: '2px' }}
+            avatar={
+              <Badge
+                anchorOrigin={{
+                  vertical: 'top',
+                  horizontal: 'right',
+                }}
+              >
+                <Avatar
+                  alt={participant.fullName}
+                  src="1.jpg"
+                  sx={{ width: 32, height: 32, fontSize: '16px' }}
+                />
+              </Badge>
+            }
+            label={participant.fullName}
+            variant="filled"
+          />
+        </>
+      )
+    }
+
+    return (
+      <Avatar
+        alt={participant.fullName}
+        src="1.jpg"
+        sx={{ width: 32, height: 32, fontSize: '16px' }}
+      />
+    )
+  })
+
+  return (
+    <AvatarGroup
+      max={6}
+      sx={{
+        margin: '18px 0 8px 7px',
```

Comment:
Maybe we could change the approach here and wrap the MUI components with `styled-components`. This way we will have a better separation between javascript and css

Reply:
We could at least separate this whole object of styles into a variable outside the component

Reply:
I'm still determining about using Styled components in this case; we can discuss it more. Still, I agree about separating the scope between CSS and JS, so to solve it now, I decided to move all the styles inside the sx props to a new location using simple objects. With that, our files are already clean than before.